CMakeLists.txt

```diff
@@ -1190,6 +1190,7 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/util/safelywritablefile.cpp
   src/util/sample.cpp
   src/util/sandbox.cpp
+  src/util/sandboxios.cpp
```

Comment:
Can we move this src file to conditional list so it is only added to the list on iOS platform?